File: ex02/C/ex02.c
#include <stdio.h>

size_t strlen(char* pointer);

int main(int argc, char* argv[])
{
    char str[20] = {0};

    printf("What is the input String? ");
    scanf("%s", str);

    printf("%s has %d chracters.", str, strlen(str));
    
    return 0;
}


size_t strlen(char* pointer)
{
    size_t returnValue = 0;

    while(pointer[returnValue] != '\0')
    {
       returnValue++; 
    }

    return returnValue;
}

File: ex02/Java/Ex02.java
import java.io.InputStream;
import java.util.Scanner;

class Ex02 {
    public static void main(String[] args) {
        String inputStr = "";

        Scanner scanner = new Scanner(System.in);

        System.out.print("What is the input String? ");

        inputStr = scanner.next();
        scanner.close();

        System.out.printf("%s has %d chracters.", inputStr, new Ex02().strlen(inputStr));
    }

    public int strlen (String str) {
        int returnValue =  0;
        
        returnValue = str.toCharArray().length;

        return returnValue;
    }
}